lib/components/form/call-taker/advanced-options.js

```diff
@@ -186,21 +193,33 @@ class AdvancedOptions extends Component {
             justifyContent: 'space-between'
           }}
         >
-          <label style={{ fontWeight: '400' }}>
-            <FormattedMessage
-              id="components.AdvancedOptions.maxWalk"
-              values={{ unitsString }}
+          {usingOtp2 ? (
+            <SliderSelector
+              label={
+                <FormattedMessage id="components.AdvancedOptions.walkReluctance" />
```

Comment:
Need to use `intl.formatMessage` so that the `aria-label` is a real string instead of a react element.

Reply:
Very good catch thanks!